src/Redux/actionCreators/UserActions.js

```diff
@@ -129,3 +133,97 @@ export const checkWalletStatus = (dispatch, getState) => {
       }
     });
 };
+
+const userDeleted = ({ history, route }) => dispatch => {
+  history.push(route);
```

Comment:
You could do this once all the other actions are dispatched. This way you have the new state when you are rendering the new route.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/17/commits/e170040f69c19fde0491348b5d7bfafe2860714d)